packages/modal/src/ModalCover.tsx

```diff
@@ -0,0 +1,21 @@
+import { forwardRef, ReactNode, HTMLAttributes } from 'react';
```

Comment:
I think we can simplify this component

- We don't need forwardRef
- We don't need to spread html attributes